Flow.Launcher.Infrastructure/UserSettings/Settings.cs

```diff
@@ -100,8 +98,12 @@ public bool HideNotifyIcon
 
         public HttpProxy Proxy { get; set; } = new HttpProxy();
 
-        [JsonConverter(typeof(StringEnumConverter))]
+        [JsonConverter(typeof(JsonStringEnumConverter))]
         public LastQueryMode LastQueryMode { get; set; } = LastQueryMode.Selected;
+
+
+        // Order defaults to 0 or -1, so 1 will let this property appear last
```

Comment:
since this comment is no longer relevant. lets replace it with 
'This needs to be loaded last by staying at the bottom'

Reply:
ok